tests/integrationv2/test_client_authentication.py

```diff
@@ -231,3 +240,53 @@ def test_client_auth_with_s2n_client_with_cert(managed_process, provider, other_
         assert b'read client certificate' in results.stderr
         assert b'read certificate verify' in results.stderr
         assert_openssl_handshake_complete(results)
+
+
+"""
+A s2n server that requests for a client certificate, and doesn't support RSA PSS,
+should downgrade to TLS 1.2 in case the client attempts to send an RSA PSS Certificate.     
```

Comment:
I think this comment can be slightly more descriptive. The triggers for this downgrade are that the server doesn't support the RSA PSS signature scheme(which happens if it builds with OpenSSL 1.0.2) and also requests client auth. The downgrade is necessary because if the client sends an RSA certificate in TLS1.3, it is mandatory that the signature scheme used be RSA PSS.